src/modules/jcms_migrate/src/Plugin/migrate/process/JCMSSplitContent.php

```diff
@@ -253,6 +243,7 @@ public function DOMnodeInnerHTML($element) {
       foreach ($element as $child) {
         $html = $child->ownerDocument->saveHTML($child);
         $html = str_replace('&nbsp;', ' ', $html);
+        $html = check_markup($html, 'basic_html');
```

Comment:
@jamiehollern The check_markup function is not available when the unit tests are run, what approach should I take to resolve this issue?